pages/board/index.tsx

```diff
@@ -1,11 +1,24 @@
 import { useQuery } from '@apollo/client';
+import { Row } from 'react-bootstrap';
 import tasksQuery from './tasks';
 
 const Board = () => {
   const { data, loading } = useQuery(tasksQuery, {
```

Comment:
Should we use the `loading` variable to show some Loading component or we don't care about it for now?

Reply:
Yes it will be used in the board section component which i am currently working on